### ISSUE TITLE
deeplab optim_parameters not called. same lr rate for all layers

### ISSUE THREAD

**Issue by gsujan** [a514514772/DISE-Domain-Invariant-Structure-Extraction#4]
Hi 

You have two different learning rates for the shared encoder( deeplab model) in the optim params function in model.py

`def optim_parameters(self, learning_rate):
        return [{'params': self.get_1x_lr_params_NOscale(), 'lr': 1 * learning_rate},
                  {'params': self.get_10x_lr_params(), 'lr': 10 * `learning_rate}]`

But this function is not called during the optimizer initilzation and you load all parameters with one learning rate. 

This is also different to the AdaptSeg code. 

Is this on purpose ? Is this giving better results than using the seperate learning rates for layer1 to layer4 and a different one for layer5 and layer6

**Comment by howard-mahe** [comment]
Hi, I don't feel this is different from AdaptSeg code.
During optimizer initialization, optim_parameters is called [here](https://github.com/a514514772/DISE-Domain-Invariant-Structure-Extraction/blob/17c587081955d528dc0a0a3b89656793d89cb428/train_dise_gta2city.py#L150).
During training, the learning is set by [adjust_learning_rate](https://github.com/a514514772/DISE-Domain-Invariant-Structure-Extraction/blob/17c587081955d528dc0a0a3b89656793d89cb428/util/utils.py#L21) function.

**Comment by gsujan** [comment]
sorry for the late reply. You are right. Closing the issue.
